enterprise/server/remote_execution/containers/docker/docker.go

```diff
@@ -160,50 +173,36 @@ func (r *dockerCommandContainer) Run(ctx context.Context, command *repb.Command,
 		}()
 	}()
 
-	r.copyContainerLogs(ctx, cid, result)
-	if result.Error != nil {
+	var stdout, stderr bytes.Buffer
+	eg := &errgroup.Group{}
+	eg.Go(func() error {
+		_, errCopy := stdcopy.StdCopy(&stdout, &stderr, hijackedResp.Reader)
+		return wrapDockerErr(errCopy, "failed to copy docker container output")
```

Comment:
nit: `err` is slightly more conventional here since there are no other error variables in scope.

```suggestion
		_, err := stdcopy.StdCopy(&stdout, &stderr, hijackedResp.Reader)
		return wrapDockerErr(err, "failed to copy docker container output")
```